relayer-cli/src/commands/tx/connection.rs

```diff
@@ -175,3 +175,167 @@ impl Runnable for TxRawConnTryCmd {
         }
     }
 }
+
+#[derive(Clone, Command, Debug, Options)]
+pub struct TxRawConnAckCmd {
```

Comment:
The four structs `TxRawConnInitCmd`, `TxRawConnTryCmd`, `TxRawConnAckCmd` and `TxRawConnConfirmCmd` are identical. Not sure if it's a good idea to try to refactor here, though, just pointing this out. We can also do it later.

Reply:
Yes, they are ..almost. The init one is again different. I looked at ways to refactor the other three, but couldn't find a good way...unless i make the actual datagram type a parameter instead of subcommand.
I think overall the CLIs would need refactoring. But these are just mainly used for development and debugging at this point.